modules/network-observability-without-loki.adoc

```diff
@@ -4,16 +4,24 @@
 :_mod-docs-content-type: REFERENCE
 [id="network-observability-without-loki_{context}"]
 = Network Observability without Loki
-You can use Network Observability without Loki by not performing the Loki installation steps and skipping directly to "Installing the Network Observability Operator". If you only want to export flows to a Kafka consumer or IPFIX collector, or you only need dashboard metrics, then you do not need to install Loki or provide storage for Loki.  Without Loki, there is no *Network Traffic* panel under *Observe*, which means there is no overview charts, flow table, or topology. The following table compares available features with and without Loki:
+You can use Network Observability without Loki by not performing the Loki installation steps and skipping directly to "Installing the Network Observability Operator". If you only want to export flows to a Kafka consumer or IPFIX collector, or you only need dashboard metrics, then you do not need to install Loki or provide storage for Loki.  Without Loki, there is no *Network Traffic* panel under *Observe*, which means there is no overview charts, flow table, or topology. The following table compares available features with and without Loki.
+
+[IMPORTANT]
+====
+Without Loki, there are some limitations for filtering and aggregations in the Network Traffic view components. For example, it is not possible to show resource-based aggregation (pods), but you can show owners (workloads).
+====
 
 .Comparison of feature availability with and without Loki
 [options="header"]
 |===
 |                                     | *With Loki* | *Without Loki*
-| *Exporters*                         | image:check-solid.png[,10]  | image:check-solid.png[,10]
-| *Flow-based metrics and dashboards*             | image:check-solid.png[,10] | image:check-solid.png[,10]
-| *Traffic Flow Overview, Table and Topology views* | image:check-solid.png[,10] | image:x-solid.png[,10]
-| *Quick Filters*                     | image:check-solid.png[,10] | image:x-solid.png[,10]
+| *Exporters*                         | image:check-solid.png[,10] | image:check-solid.png[,10]
+| *Flow-based metrics and dashboards* | image:check-solid.png[,10] | image:check-solid.png[,10]
+| *Traffic Flows Overview* ^[1]^      | image:check-solid.png[,10] | image:check-solid.png[,10]
+| *Network Traffic Table* ^[1]^       | image:check-solid.png[,10] | image:x-solid.png[,10]
+| *Topology view* ^[1]^               | image:check-solid.png[,10] | image:check-solid.png[,10]
+| *Quick Filters* ^[1]^               | image:check-solid.png[,10] | image:x-solid.png[,10]
```

Comment:
Quick filters are possible without Loki too
TBH I'm not sure if the feature is "big enough" to mention it here, perhaps we can remove it

Reply:
Removed :)